src/api/EscargotPublic.cpp

```diff
@@ -1297,6 +1297,158 @@ void DebuggerOperationsRef::BreakpointOperations::getLexicalScopeChain(uint32_t
     }
 }
 
+static void fillObjectProperties(ExecutionState* state, Object* object, Object::OwnPropertyKeyVector& keys, DebuggerOperationsRef::PropertyKeyValueVector& result, size_t start)
+{
+    size_t size = keys.size();
+
+    for (size_t i = 0; i < size; i++) {
+        ObjectPropertyName propertyName(*state, keys[i]);
+
+        result[start + i].key = toRef(keys[i].toStringWithoutException(*state));
+
+        try {
+            ObjectGetResult value = object->getOwnProperty(*state, propertyName);
+
+            result[start + i].value = OptionalRef<ValueRef>(toRef(value.value(*state, Value(object))));
```

Comment:
IMO result of `getOwnProperty` always has valid value (return undefined in default), so I think that `OptionalRef<ValueRef>` is unnecessary here.